app.py

```diff
@@ -6,7 +6,7 @@
 from werkzeug import secure_filename
 
 # These are the extension that we are accepting to be uploaded
-ALLOWED_EXTENSIONS = set(['png', 'jpg', 'jpeg'])
+ALLOWED_EXTENSIONS = set(['png', 'jpg', 'jpeg','svg'])
```

Comment:
you can use it again for logo part as SVG for wallpaper doesn't seem right

Reply:
Yeah, working on it @xeon-zolt

Reply:
yeah @xeon-zolt on it